auslib/admin/views/releases.py

```diff
@@ -507,3 +507,22 @@ def _post(self, changed_by, transaction):
         )
         response.headers['Content-Type'] = 'application/json'
         return response
+
+
+class SingleReleaseColumnView(AdminView):
+    """ /releases/column/:column"""
+
+    def get(self, column):
+        releases = dbo.releases.getReleases()
```

Comment:
This is an extremely expensive query, and should be replaced with getReleaseInfo(). The main difference between those two methods is that the latter will return Release.data columns, which are extremely big, and not something you could filter on anyways.